formatters/formatter.py

```diff
@@ -0,0 +1,2 @@
+def json_user_object(user):
+    return {"email": user.email, "name": user.name, "last_name": user.last_name, "phone": user.phone, "gender": user.gender}
```

Comment:
we should format this multiline for better readability  as a sent you in my slack msg